tests/multiple_listening_addresses.rs

```diff
@@ -30,22 +30,17 @@ fn multiple_concurrent_ephemeral_listening_addresses_on_same_ip() {
 
         let (first, second) = futures::future::join(first, second).await;
 
-        // before an Swarm alike api on the background task to make sure we attempt to modify the
-        // background task twice before the swarm gets to poll, this will forever produce one or two
-        // successes.
+        // before we have an Swarm-alike api on the background task to make sure the two futures
```

Comment:
nit(grammar): a~~n~~ Swarm-~~a~~like api **i**n the background

Reply:
also wording: "...the background task**, in order** to make sure the two futures ~~would~~ attempt to modify **that task** before a poll to the inner swarm, this ~~will~~ **has to** produce one or two successes.